src/main/routes.ts

```diff
@@ -50,7 +51,11 @@ export class Routes {
       app.get(step.url, errorHandler(controller.get));
 
       if (step.form) {
-        app.post(step.url, errorHandler(new PostController(new Form(step.form)).post));
+        if (jurisdictionSteps.map(String).includes(step.url)) {
```

Comment:
I'll tidy this up in another PR so the step scanner detects whether there is a custom post controller defined.